### DISCUSSION TITLE
Pieces by LEGO part number

### DISCUSSION THREAD

**Discussion by ambs** [leozide/leocad#822] [General]
Hi
I am new to LeoCad, and also new to GitHub Discussions, so sorry if this is not the right place, or if this question is a duplicate.
I was trying to find a part, using its lego part number (4656764). I did not found it. But then, searching by "car" I saw a few car base plates, and therefore, gone one by one looking for the right one.

Is there any mapping, or any way we can use the part number to get the right part?

Thanks

**Comment by rsbx** [comment]
LeoCAD uses LDraw.org naming conventions; therefor, LDraw part numbers 
are "design IDs" AKA shape ID. "4656764" is not a design ID but appears 
to be a LEGO (manufacturing?) ID which includes color and, possibly, 
mold variant information.

An online search for "4656764" led me to brickset.com which identified 
the as having design ID "30029" (shape only).

Since it seems that you're working from LEGO instructions, either ignore 
the LEGO part ID and search for the shape in the parts catalog, or do an 
online search for the LEGO part ID to find the design ID.

**Reply by ambs** [reply]
Hi.
Thanks for the answer.
As I am not a native English speaker, sometimes it gets difficult to know how to search for a part. Looking in the parts catalogue at the end of the official booklets allow me to see that ID (that as you mention, might include color as well).
If there is no way to do that kind of search inside LeoCad, I will use your suggestion searching in the web (namely brickset.com).

Cheers

**Comment by rsbx** [comment]
Sometimes it requires research Google/DuckDuckGO/... or the patience to 
look through all the shapes in the local catalog. In this instance, 
brickset.com was the first listed result to my search. Another useful 
site for part or set research is bricklink.com.
